### PR TITLE
release note for Android Core 1.5.0

### PR DESCRIPTION
**Pull request by shalehaha**

### CODE REVIEW

File: release-notes.md [hunk 1]
```diff
@@ -8,6 +8,21 @@ description: Release notes and change logs for the Adobe Experience Platform Mob
...
+The following updates were made in this release:
+
+**Android Core 1.5.0**
+
+* Fixed a cusor leak.
+* Fixed an issue where the advertising identifier was duplicated in the response to the `MobileCore.getSDKIdentifiers` API.
+* Added support for overriding internal network stack with customer-provided code.
+* Added new a interface with failure callback, `AdobeCallbackWithError`, which can be used with the methods `MobileCore.getPrivacyStatus`,  `MobileCore.getSdkIdentities`. And we will gradually add the support in other extensions to enable the failure callback.
+
+**Android Mobile Servcies 1.1.1**
```
Comment (emdobrin) [inline release-notes.md]: Typo: Mobile Services

File: release-notes.md [hunk 2]
```diff
@@ -8,6 +8,21 @@ description: Release notes and change logs for the Adobe Experience Platform Mob
 
 Please take a moment to fill out a [short survey](https://www.surveymonkey.com/r/AEPDocs) on how we can better assist you with enabling Adobe Experience Cloud solutions and services on your mobile apps.
 
+## February 4, 2020
+
+The following updates were made in this release:
+
+**Android Core 1.5.0**
```
Comment (emdobrin) [inline release-notes.md]: Please include these additions in each extension release notes page.

File: using-mobile-extensions/mobile-core/mobile-core-release-notes.md [hunk 3]
```diff
@@ -7,14 +7,11 @@ The following updates were made in this release:
...
-
 * Fixed an issue where the advertising identifier was duplicated in the response to the `MobileCore.getSDKIdentifiers` API.
-
 * Added support for overriding internal network stack with customer-provided code.
+* Added a new interface with failure callback, `AdobeCallbackWithError`, which can be used with the `MobileCore.getPrivacyStatus`,  `MobileCore.getSdkIdentities ` methods. 
 
-* Added new a interface with failure callback, `AdobeCallbackWithError`, which can be used with the methods `MobileCore.getPrivacyStatus`,  `MobileCore.getSdkIdentities`. And we will gradually add the support in other extensions to enable the failure callback.
-
-  
+​       We plan to gradually add the ability to enable failure callback to the other extensions.
```
Comment (rekrishn) [inline using-mobile-extensions/mobile-core/mobile-core-release-notes.md]: What I meant was this:
"* Added a new interface with failure callback, AdobeCallbackWithError, which can be used with the MobileCore.getPrivacyStatus, MobileCore.getSdkIdentities methods.
​     We plan to gradually add the ability to enable failure callback to the other extensions."
where "We" should line up with "Added". Hope that is clearer!

File: release-notes.md [hunk 4]
```diff
@@ -17,7 +17,9 @@ The following updates were made in this release:
 * Fixed a cursor leak.
 * Fixed an issue where the advertising identifier was duplicated in the response to the `MobileCore.getSDKIdentifiers` API.
 * Added support for overriding internal network stack with customer-provided code.
-* Added new a interface with failure callback, `AdobeCallbackWithError`, which can be used with the methods `MobileCore.getPrivacyStatus`,  `MobileCore.getSdkIdentities`. And we will gradually add the support in other extensions to enable the failure callback.
+* Added a new interface with failure callback, `AdobeCallbackWithError`, which can be used with the `MobileCore.getPrivacyStatus`,  `MobileCore.getSdkIdentities ` methods. 
+
+​       We plan to gradually add the ability to enable failure callback to the other extensions.
```
Comment (rekrishn) [inline release-notes.md]: What I meant was this:
"* Added a new interface with failure callback, AdobeCallbackWithError, which can be used with the MobileCore.getPrivacyStatus, MobileCore.getSdkIdentities methods.
​ We plan to gradually add the ability to enable failure callback to the other extensions."
where "We" should line up with "Added". Hope that is clearer!

**Pull request merged**